<launch>
  <node name="grouper_dynamixel_control" pkg="dynamixel_control" type="dynamixel_control" required="true" output="screen">
    <rosparam>
      device_name_: "/dev/ttyUSB0"
      baud_rate_: 57600
      protocol_version_: 1.0
      motor_count_: 3
      motor_0_:
        motor_name_: "cam_pan"
        motor_model_: "RX_10"
        motor_id_: 2
      motor_1_:
        motor_name_: "cam_tilt"
        motor_model_: "RX_10"
        motor_id_: 3
      motor_2_:
        motor_name_: "radar_tilt"
        motor_model_: "MX_106"
        motor_id_: 1
   </rosparam>

  </node>
  <node name="urg_grouper" pkg="urg_node" type="urg_node" required="true" output="screen">
    <rosparam>
      ip_address: "192.168.0.10"
      ip_port: 10940
      publish_intensity: true
      publish_multiecho: false
    </rosparam>
  </node>

  <node name ="grouper_lidar" pkg="grouper_lidar" type="grouper_lidar"/>

  <node pkg="rqt_rviz" type="rqt_rviz" name="rqt_rviz" args="-d $(find grouper_lidar)/config/grouper.rviz"/>

</launch>
